lib/src/header_parser/utils.dart

```diff
@@ -89,14 +89,14 @@ extension CXCursorExt on clang_types.CXCursor {
     return s;
   }
 
-  /// Dispose type using [type.dispose].
```

Comment:
I think this is a mistake in the documentation. Doesn't look like it's necessary (or possible) to dispose a `CXType`